java/google/registry/rde/imports/XjcToDomainResourceConverter.java

```diff
@@ -58,9 +62,25 @@
 import google.registry.xjc.secdns.XjcSecdnsDsDataType;
 import org.joda.time.DateTime;
 
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Random;
+
 /** Utility class that converts an {@link XjcRdeDomainElement} into a {@link DomainResource}. */
 final class XjcToDomainResourceConverter extends XjcToEppResourceConverter {
 
+  private static final RandomStringGenerator stringGenerator = new RandomStringGenerator(
+      StringGenerator.Alphabets.BASE_64, getRandom());
+
+  private static Random getRandom() {
+    try {
+      return SecureRandom.getInstance("NativePRNG");
+    } catch (NoSuchAlgorithmException e) {
+      // In the case that secure random isn't available, use insecure random instead.
```

Comment:
Since this Random gets assigned to a static final field, I have a choice between throwing an unchecked exception or using a default Random instance. Is an unchecked exception preferable?